tests/test-suite.test.ts

```diff
@@ -80,6 +80,22 @@ describe('Passage API Requests', () => {
             const user = await passage.user.get(userID);
             expect(user).toHaveProperty('id', userID);
         });
+        test('getUserByIdentifier', async () => {
+            const randomEmail = `${Math.random().toString(36).substr(2, 20)}@gmail.com`;
```

Comment:
sonar cloud is failing bc of `math.random` in this test. I think this is acceptable but open to suggestions for changing it.

Reply:
Acceptable here. I marked it as safe an re-ran.